tutorials/Coordinates/Coordinates.ipynb

```diff
@@ -0,0 +1,781 @@
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "coo_sdss = SkyCoord(sdss['ra']*u.deg, sdss['dec']*u.deg)\n",
+      "coo_twomass = SkyCoord(twomass['ra'], twomass['dec'])"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": []
+    },
+    {
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "Note a subtle difference here: you had to give units for SDSS but *not* for 2MASS.  This is because the 2MASS table has units associated with the columns, while the SDSS table does not (so you have to put them in manually).\n",
+      "\n",
+      "Now we simply use the ``SkyCoord.match_to_catalog_sky`` method to match the two catalogs. Note that order matters: we're matching 2MASS to SDSS because there are many *more* entires in the SDSS, so it seems likely that most 2MASS objects are in SDSS (but not vice versa)."
+     ]
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "idxsdss, d2dsdss, d3dsdss = coo_twomass.match_to_catalog_sky(coo_sdss)"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": []
+    },
+    {
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "``idx`` is the indecies into ``coo_sdss`` that get the closest matches, while ``d2d`` and ``d3d`` are the on-sky and real-space distances between the matches. In our case ``d3d`` can be ignored because we didn't give a line-of-sight distance, so its value is not particularly useful.   But ``d2d`` provides a good diagnosis of whether we actually have real matches:"
+     ]
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "plt.hist(d2dsdss.arcsec,histtype='step', range=(0,2))\n",
+      "plt.xlabel('separation [arcsec]')\n",
+      "plt.tight_layout()"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": []
+    },
+    {
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "Ok, they're all within an arcsecond that's promising.  But are we sure it's not just that *anything* has matches within an arcescond?  Lets check by putting a grid of random points down and comparing them to the SDSS catalog."
+     ]
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "np.random.rand(len(coo_twomass))*coo_sdss.ra.ptp() + coo_sdss.ra.min()"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": []
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "sim_ras = np.random.rand(len(coo_twomass))*coo_sdss.ra.ptp() + coo_sdss.ra.min()\n",
+      "sim_decs = np.random.rand(len(coo_twomass))*coo_sdss.dec.ptp() + coo_sdss.dec.min()\n",
+      "# don't need units explicitly because we got them from the multiplication\n",
+      "coo_simulated = SkyCoord(sim_ras, sim_decs)  \n",
+      "idx, d2d, d3d = coo_simulated.match_to_catalog_sky(coo_sdss)"
+     ],
```

Comment:
Hmm, my thinking here is that this reaches the point that it's ok to assume the reader has some sense of why this might be useful.  Definitely would be good to get some other opinions on this, though...